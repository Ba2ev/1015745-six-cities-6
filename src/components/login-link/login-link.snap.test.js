import React from 'react';
import {render} from '@testing-library/react';
import {Router} from 'react-router-dom';
import {createMemoryHistory} from 'history';
import LoginLink from './login-link';

it(`LoginLink should render correctly`, () => {
  const history = createMemoryHistory();

  const {container} = render(
      <Router history={history}>
        <LoginLink />
      </Router>
  );

  expect(container).toMatchSnapshot();
});
