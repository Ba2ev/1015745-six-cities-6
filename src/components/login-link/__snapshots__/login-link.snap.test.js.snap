// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginLink should render correctly 1`] = `
<div>
  <a
    class="header__nav-link header__nav-link--profile"
    href="/login"
  >
    <div
      class="header__avatar-wrapper user__avatar-wrapper"
    />
    <span
      class="header__login"
    >
      Sign in
    </span>
  </a>
</div>
`;
